CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 ### Internal
 
 - Update docs: configuration of routes and addonRoutes @ksuess
+- Translation de: aria-label "Contents". "Inhalte" not "Inhaltsverzeichnis" @ksuess
```

Comment:
@ksuess nitpicking, no blocker for a merge: this is a bugfix for me. Internal refers to tooling etc. Your PR fixes a problem and improves the accessibility of Volto.